mythril/interfaces/cli.py

```diff
@@ -42,6 +47,19 @@ def main():
     parser = argparse.ArgumentParser(
         description="Security analysis of Ethereum smart contracts"
     )
+    create_parser(parser)
+
+    # Get config values
+
+    args = parser.parse_args()
+    parse_args(parser=parser, args=args)
+
+
+if __name__ == "__main__":
+    main()
+
+
+def create_parser(parser):
```

Comment:
Since we touched it, can we get docstrings here and for `parse_args`? And type hints if appropriate. `exit_with_error` can do with some type hints as well.